### ISSUE TITLE
Could it show lastest ten images?

### ISSUE THREAD

**Issue by SRadiant** [thomasjacquin/allsky-website#13]
Hi, Thomas.
I want to show the lastest ten images (small size) below the liveview image, and can be click to show large size image.
(Like the photo show)
![123](https://user-images.githubusercontent.com/46018730/65113520-10f2e400-da16-11e9-8343-6c9d2c1087a9.png)
I have no idea how to do it, could you help me?
Thanks a lot.

**Comment by thomasjacquin** [comment]
This is technically doable but it would require quite a bit of changes. One of the reason why is that the camera is only uploading the same image at regular intervals and there is no timestamp in the filename.

Also you would need some code on your host that looks for the latest 10 images (assuming you're not overwriting image-resize.jpg everytime) and displays them on screen. That could be done with PHP. You will also need some javascript code to refresh the images at regular interval.

So to summarize, yes it's possible but it take some time to implement the solution.
